src/pages/Components/GridCardDisplay/GridDisplay.module.css

```diff
@@ -0,0 +1,50 @@
+.gridcont {
+    display: grid;
+    grid-template-areas: 
+    "a a a d d e e g"
+    "a a a d d e e g"
+    "a a a b b b b g"
+    "a a a b b b b g"
+    "c c c c c f f f"
+    "c c c c c f f f"
+    ;
```

Comment:
nice that you are using grid areas but instead of using letters give the areas actual names

Reply:
Oh yeah thanks, I was testing it that time.